packages/components/src/CheckableTag/CheckableTag.test.tsx

```diff
@@ -1,41 +1,161 @@
-import { vi } from 'vitest';
 import { useEffect, useRef } from 'react';
-import { renderWithTheme, screen, userEvents } from '@astral/tests';
+import {
+  describe,
+  expect,
+  it,
+  renderWithTheme,
+  screen,
+  userEvents,
+  vi,
+} from '@astral/tests';
 
 import { CheckableTag } from './CheckableTag';
+import { type CheckableTagAddon } from './types';
 
 describe('CheckableTag', () => {
   it('Ref доступен', () => {
-    const resultRef = { current: null };
+    const resultRefMock = { current: null };
 
     const CheckableTagWithRef = () => {
       const ref = useRef(null);
 
       useEffect(() => {
-        resultRef.current = ref.current;
+        resultRefMock.current = ref.current;
       }, []);
 
       return <CheckableTag ref={ref} />;
     };
 
     renderWithTheme(<CheckableTagWithRef />);
-    expect(resultRef?.current).not.toBeNull();
+    expect(resultRefMock.current).not.toBeNull();
   });
 
-  it('Тэг блокируется при disabled=true', () => {
-    renderWithTheme(<CheckableTag disabled />);
+  it('Callback Ref доступен', () => {
+    const callbackRefSpy = vi.fn();
 
-    const tag = screen.getByRole('checkbox', { hidden: true });
+    const sut = <CheckableTag ref={callbackRefSpy} />;
 
-    expect(tag).toBeDisabled();
+    renderWithTheme(sut);
+    expect(callbackRefSpy).toBeCalledWith(expect.any(HTMLElement));
   });
 
-  it('onChange вызывается при нажатии на тэг', async () => {
+  it('Компонент получает checked состояние при передаче checked=true через props', () => {
```

Comment:
```suggestion
  it('Компонент получает checked состояние если checked=true', () => {
```

Так будет короче, смысл не потерялся